packages/contracts/contracts/libs/v0.8.x/minter-libs/SettlementExpLib.sol

```diff
@@ -339,8 +381,64 @@ library SettlementExpLib {
                 _projectId,
                 _coreContract
             );
+
+        // if this is the first purchase on this minter, set the number of
+        // of tokens to be auctioned to:
+        // (minter max invocations) - (current core contract invocations)
+        if (settlementAuctionProjectConfig.numPurchasesOnMinter == 0) {
+            // get up-to-data invocation data from core contract
+            (
+                uint256 coreInvocations,
+                uint256 coreMaxInvocations
+            ) = MaxInvocationsLib.coreContractInvocationData(
+                    _projectId,
+                    _coreContract
+                );
+            // snap chalkline on the number of tokens to be auctioned on this
+            // minter
+            // @dev ackgnowledge that this value may be stale if the core
```

Comment:
sp: acknowledge

Reply:
good call, acknowledged and fixed in [3f96e49](https://github.com/ArtBlocks/artblocks-contracts/pull/1091/commits/3f96e4907fc321d93c96a3368c1e253d08970a01)